waspc/e2e-test/test-outputs/waspMigrate-golden/waspMigrate/.wasp/out/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-alpine AS node
+FROM node:18-alpine AS node
```

Comment:
Same question, no minor here?

Reply:
Ditto to CI answer. I believe it should always be `18.12.0` or higher from now on, and always stay as LTS in here: https://github.com/nodejs/docker-node/blob/f05fbf88068ba29ac9b544a72c9471ba60243e4b/18/alpine3.16/Dockerfile